content/developer-platform/tutorials/your-first-service/_index.en.md

```diff
@@ -37,9 +83,16 @@ the Service.
 Once the service is created, KDP will provision a kubeconfig. This kubeconfig is meant to be used
 by the KDP Servlet, the component installed by service owners into the service cluster.
 
-(TODO: Make this kubeconfig available via the KDP Dashboard.) You can also use `kubectl` to navigate
-into your workspace and inspect your Service object. In `spec.kubeconfig` you will find the name of
-the kubeconfig Secret that you can use for your Servlet.
+The generated kubeconfig is available from the service's context menu in the Dashboard.
+
+![Service Context Menu](/img/developer-platform/common/service-context-menu.png)
+
+Click "kubeconfig" to inspect and download the kubeconfig:
+
+![Kubeconfig View](/img/developer-platform/common/kubeconfig-view.png)
```

Comment:
Please do not put images in static/img/, we _just_ undid all this stuff and got rid of the "common" (but not actually common) images.

Reply:
cf. https://github.com/kubermatic/docs/pull/1654

Reply:
Forgot about that.